docs/rotation_schedule_extend.md

```diff
@@ -0,0 +1,36 @@
+## rotation schedule extend
+
+Extends a previously generated schedule
+
+### Synopsis
+
+Extends a previously generated schedule
```

Comment:
So, these files seem to be auto generated. (I'm looking at the h5 at the end of each of these. Does that mean the auto-gen functionality forces this format (h2 name of command, then sentence, then h3 synopsis + the same sentence)?

Reply:
Yes, these are autogenerated. It looks like the [format is hardcoded](https://github.com/spf13/cobra/blob/master/doc/md_docs.go#L54), but to your point about repetition it look like I can provide a `Long` value which should reduce the duplication.